Documentation/parameters.par

```diff
@@ -92,7 +92,9 @@ CFL = 0.5                            ! CFL number (<=1.0)
 FixTimeStep = 5                      ! Manually chosen maximum time step
 ClusteredLTS = 2                     ! 1 for Global time stepping, 2,3,5,... Local time stepping (advised value 2)
 !ClusteredLTS defines the multi-rate for the time steps of the clusters 2 for Local time stepping
-LtsWeightTypeId = 1                  ! 0=exponential, 1=exponential-balanced, 2=encoded
+NodeWeightModelTypeId = 0            ! 0 for Exponential, 1 for Exponential Balanced, 2 for Encoded Balanced
+EdgeWeightModelTypeId = 0            ! 0 for Naive, 1 for Communication Approximation, 2 for 1 + Minimum Messaging, 3 for 1 + Balanced Messaging
```

Comment:
Hi actually adding something like that in mind, but I thought it could be too long. (I had added a runtime check for that, but if you suggest then I also see no problem adding it)

Reply:
There is also another thing I remembered the I should actually write something like:

NodeWeightModelTypeId = 0            ! 0 for Exponential, 1 for Exponential Balanced, 2 for Encoded Balanced, 3 for 1 + Minimum Messaging, 4 for 1 + Balanced Messaging
EdgeWeightModelTypeId = 0            ! 0 for Naive, 1 for Communication Approximation,
                                     ! Strategies 3 and 4 require the edge weight strategy to be set to 1 

as the strategies are actually bound to vertex weight. otherwise it will confuse the user

Reply:
I think it will be clearer to the user, (and briefly forgot they were bound to nodeweightmodeltypeid)

Reply:
yes